### DISCUSSION TITLE
Github issues are automatically disabled

### DISCUSSION THREAD

**Discussion by troy-system76** [community/community#22524] [New to GitHub]
<p>We have a private repository. I enabled Issues for that repo by going into Settings and then enabling them under Features. However the next day I found out Issues were disabled. This has happened a few days in a row now. Nobody I know of is disabling the feature and logs don’t show this kind of behavior. Does anybody know what could be causing this? What additional information can I supply?</p>
<p>Thanks in advance!</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>I’m enabling and disabling one of my private repos Issues tab right now, and it’s working,</p>
<p>try to repeat that again,</p>
<p>who else has write access to your repo? they can change it too,</p>

**Comment by troy-system76** [comment]
<p>Thanks <a class="mention" href="/jdevstatic">@jdevstatic</a>. You’re right. We had a terraform process overriding the setting. Thanks for the help!</p>
